Program.cs

```diff
@@ -145,162 +145,145 @@ private static void HandlingOfMenu()
                 ShowMenu();
             }
         }
-
-        private static void SelectPatient()
+        private static int SelectAnimal()
         {
             Console.WriteLine($"Lista pacjentów: {Environment.NewLine} 1.Psy: {Environment.NewLine} 2.Koty: ");
             int selectAnimal = int.Parse(Console.ReadLine());

```

Comment:
You should find some way for difference in variables and method names. For example Select Animal for function, is ok - its verb, but selectAnimal for variable is not correct - more apt will be selectedAnimal, becouse this thing is done.  Then if you want to search selectedAnimal references You get only things thats interest You.